### PR TITLE
RDK-30526, RDK-30475: added client control westeros extension

### PR DESCRIPTION
**Pull request by mateusz-hobgarski-red**

### CODE REVIEW

**Comment by mhughesacn** [comment]
Hi Mateusz - I fixed the Blackduck errors (spurious pattern detected in the source code), and you have removed another match before I got to it, so all the compliance checks are passing.  But any new source code files should have an RDK Apache header, and some do not.  (This won't block merge but it should really be fixed.)
Thanks,
Martin
Martin Hughes, RDK CMF Compliance Team

**Comment by mateusz-hobgarski-red** [comment]
Hi Martin, I've added the missing RDK Apache headers.
Regards,
Mateusz Hobgarski

**Comment by mhughesacn** [comment]
Many thanks Mateusz

File: rdkcompositor.cpp [hunk 1]
```diff
@@ -150,33 +151,67 @@ namespace RdkShell
         }
     }
 
-    bool RdkCompositor::createDisplay(const std::string& displayName, uint32_t width, uint32_t height)
+    bool RdkCompositor::loadExtensions(WstCompositor *compositor, const std::string& clientName)
```
Comment (mfiess) [inline rdkcompositor.cpp:154]: Let's have `loadExtensions()` return true on success and false on failure to follow the consistency of the other RDKShell apis.

**Review by mfiess — CHANGES_REQUESTED** [review]

File: rdkcompositor.cpp (hunk 1 above)
Comment (mateusz-hobgarski-red) [reply rdkcompositor.cpp:154]: done

**Review by mfiess — APPROVED** [review]

**Pull request merged**